docs/notebooks/tutorial_axes_anndata_mudata.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    rna_raw=pbmc5k["rna"].copy()
</pre><p>I think it would be good to mention somewhere what copy does and when one should use it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>